### PR TITLE
py sol for close tuples

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by freakin23** [comment]
AC: https://codeforces.com/contest/1462/submission/280335128

File: solutions/gold/cf-1462E2.mdx [hunk 1]
```diff
@@ -101,4 +101,88 @@ int main() {
...
+	nums = list(map(int, input().split()))
+	freq = [0] * (n + 1)  # frequency of each element
+	preFreq = [0] * (n + 1)  # prefix-array for freq vector
+	ans = 0
+
+	for x in nums:
+		freq[x] += 1
+
+	for i in range(1, n + 1):
+		preFreq[i] = freq[i] + preFreq[i - 1]
```
Comment (SansPapyrus683) [inline solutions/gold/cf-1462E2.mdx:160]: why camelcase for this name bro

Comment (freakin23) [reply solutions/gold/cf-1462E2.mdx:160]: i was just keeping the var names same, which you only use to tell me

Comment (freakin23) [reply solutions/gold/cf-1462E2.mdx:160]: should i use snake case here

Comment (SansPapyrus683) [reply solutions/gold/cf-1462E2.mdx]: yeah
idk what the c++ version is on...

File: solutions/gold/cf-1462E2.mdx [hunk 2]
```diff
@@ -100,5 +100,84 @@ int main() {
...
+		num_larger_elems = pre_freq[max_index] - pre_freq[i]
+		"""
+		We count the total number of m-tuples with elements between i and
+		max_index. Then, we subtract the tuples that don't have i as their
+		minimum element, as they will be counted later on.
+		"""
+		curr_ans = (
+			choose(num_total_elems, m) - choose(num_larger_elems, m) + MOD
+		) % MOD
+		ans = (ans + curr_ans) % MOD
```
Comment (ryanchou-dev) [inline solutions/gold/cf-1462E2.mdx:177]: could you space out these lines to make it easier to read?

File: solutions/gold/cf-1462E2.mdx [hunk 3]
```diff
@@ -100,5 +100,84 @@ int main() {
...
+		num_total_elems = pre_freq[max_index] - pre_freq[i - 1]
+		# the number of elements from [i+1, max_index]
+		num_larger_elems = pre_freq[max_index] - pre_freq[i]
+		"""
+		We count the total number of m-tuples with elements between i and
+		max_index. Then, we subtract the tuples that don't have i as their
+		minimum element, as they will be counted later on.
+		"""
+		curr_ans = (
+			choose(num_total_elems, m) - choose(num_larger_elems, m) + MOD
```
Comment (ryanchou-dev) [inline solutions/gold/cf-1462E2.mdx:175]: this +MOD is unnecessary i think

File: solutions/gold/cf-1462E2.mdx (hunk 1 above)
Comment (freakin23) [reply solutions/gold/cf-1462E2.mdx]: snake case implemented

File: solutions/gold/cf-1462E2.mdx (hunk 2 above)
Comment (freakin23) [reply solutions/gold/cf-1462E2.mdx]: okay

File: solutions/gold/cf-1462E2.mdx (hunk 3 above)
Comment (freakin23) [reply solutions/gold/cf-1462E2.mdx]: yes

**Comment by freakin23** [comment]
AC: https://codeforces.com/contest/1462/submission/280401703

**Review by SansPapyrus683 — APPROVED** [review]
show ac ss

**Comment by freakin23** [comment]
> show ac ss

Here is the link:
https://codeforces.com/contest/1462/submission/280401703

**Review by ryanchou-dev — APPROVED** [review]

**Pull request merged**